xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -960,7 +987,9 @@ bool ROCMBlas::DoBlasGemmBatched(
     DeviceMemorySlice<std::complex<double>> b_array, int ldb,
     std::complex<double> beta, DeviceMemorySlice<std::complex<double>> c_array,
     int ldc, int batch_count, const NumericOptions &numeric_options,
-    ScratchAllocator *scratch_allocator) {
+    ScratchAllocator *scratch_allocator,
+    blas::CallContext context) {
```

Comment:
nit: I think this can go into the previous line as well without exceeding the 80 character limit

Reply:
Done